plugins/adr/README.md

```diff
@@ -71,16 +71,32 @@ Afterwards, add the following code snippet to use `AdrSearchResultListItem` when
 import { AdrSearchResultListItem } from '@backstage/plugin-adr';
 
 ...
+<SearchType.Accordion
+  name="Result Type"
+  defaultValue="software-catalog"
+  types={[
+    ...
+    {
+      value: 'adrs',
```

Comment:
If you're using the default ADR search collator, this should be `adr`: https://github.com/backstage/backstage/blob/f0ca6e0adf4566097f58484df21df65b2ab564c3/plugins/adr-backend/src/search/DefaultAdrCollatorFactory.ts#L99